News Theme Development.md

```diff
@@ -54,19 +54,19 @@ The engine-theme-sdk is located at: <https://github.com/WPMedia/engine-theme-sdk
 
 For contributing to that repo, please see its contributing guidelines.
 
-### fusion-news-theme-blocks
+### arc-themes-blocks
 
-The fusion-news-theme-blocks repo is located at: <https://github.com/WPMedia/fusion-news-theme-blocks>. In a typical client setup, you have a feature pack repository and inside you have a components directory where you store your code for feature, chains, etc. In our theme's environment, the majority of these components will be stored in this repo instead. We refer to these external components as "blocks." The components directory in the feature pack will be reserved for custom components that the client might build. This separation from the feature pack will allow the customer to pick and choose the components that they want added in their site. Also, because fusion-news-them-blocks is a multi-repo (using Lerna), any unused components will not be part of the client bundle.
+The arc-themes-blocks repo is located at: <https://github.com/WPMedia/arc-themes-blocks>. In a typical client setup, you have a feature pack repository and inside you have a components directory where you store your code for feature, chains, etc. In our theme's environment, the majority of these components will be stored in this repo instead. We refer to these external components as "blocks." The components directory in the feature pack will be reserved for custom components that the client might build. This separation from the feature pack will allow the customer to pick and choose the components that they want added in their site. Also, because fusion-news-them-blocks is a multi-repo (using Lerna), any unused components will not be part of the client bundle.
```

Comment:
```suggestion
The arc-themes-blocks repo is located at: <https://github.com/WPMedia/arc-themes-blocks>. In a typical client setup, you have a feature pack repository and inside you have a components directory where you store your code for feature, chains, etc. In our theme's environment, the majority of these components will be stored in this repo instead. We refer to these external components as "blocks." The components directory in the feature pack will be reserved for custom components that the client might build. This separation from the feature pack will allow the customer to pick and choose the components that they want added in their site. Also, because arc-themes-blocks is a multi-repo (using Lerna), any unused components will not be part of the client bundle.
```